src/apps/monitoring/iplabel/newtest/restapi/mode/scenarios.pm

```diff
@@ -299,6 +305,10 @@ Filter robots (can be a regexp).
 
 Filter scenarios (can be a regexp).
 
+=item B<--timeframe>
+
+Define timeframe in seconds (by default it's the time between two executions).
```

Comment:
Define timeframe (time between two executions) in seconds (default value: 300 ).
The first range is calculated with this value then it's calculated from the timestamp present in the cache file.